EasyPost/Models/API/ChildUserCollection.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Linq;
+using EasyPost.Models.Shared;
+using EasyPost.Parameters.User;
+using Newtonsoft.Json;
+
+namespace EasyPost.Models.API;
+
+/// <summary>
+///     Class representing a collection of EasyPost Child <see cref="User"/>s.
+/// </summary>
+public class ChildUserCollection : PaginatedCollection<User>
+{
+    #region JSON Properties
+
+    /// <summary>
+    ///     The Child <see cref="User"/>s in the collection.
+    /// </summary>
+    [JsonProperty("children")]
+    public List<User>? Children { get; set; }
```

Comment:
User and Children have different attributes, the list of users here doesn't seem correct. We should probably need to create a child model in C# lib.

Reply:
After discussion, we decided not to make a separate `ChildUser` class due to serialization issues.

However, I took the opportunity to clean up some unnecessary inheritance by dropping the `BaseUser` class.